com.amazon.Workspaces.json

```diff
@@ -0,0 +1,65 @@
+{
+  "app-id": "com.amazon.Workspaces",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "19.08",
+  "sdk": "org.freedesktop.Sdk",
+  "command": "workspacesclient",
+  "tags": [
+    "proprietary"
+  ],
+  "finish-args": [
+    "--socket=x11",
+    "--share=ipc",
+    "--device=dri",
+    "--socket=pulseaudio",
+    "--share=network"
+  ],
+  "modules": [
+    { "name": "workspacesclient",
+      "buildsystem": "simple",
+      "build-commands": [
+        "mkdir -p /app/bin /app/lib",
+        "install apply_extra /app/bin",
+        "install -Dm644 com.amazon.Workspaces.appdata.xml /app/share/appdata/com.amazon.Workspaces.appdata.xml",
+        "cp /usr/bin/ar /usr/bin/desktop-file-install /app/bin",
+        "ARCH_TRIPLE=$(gcc --print-multiarch) && cp /usr/lib/${ARCH_TRIPLE}/libbfd-*.so /app/lib",
+        "ln -s /app/extra/bin/workspacesclient /app/bin/workspacesclient"
+      ],
+      "sources": [
+        {
+          "type": "script",
+          "dest-filename": "apply_extra",
+          "commands": [
+            "ar x 'workspacesclient.deb' data.tar.xz",
+            "tar xf data.tar.xz ./usr ./opt",
+            "rm -f data.tar.xz",
+            "mv opt/workspacesclient ./bin",
+            "install -Dm644 usr/share/pixmaps/com.amazon.workspaces.svg export/share/icons/hicolor/scalable/apps/com.amazon.Workspaces.svg",
+            "install -Dm644 usr/share/applications/workspacesclient.desktop export/share/applications/com.amazon.Workspaces.desktop",
+            "rm -rf usr opt",
+            "desktop-file-install --dir=/app/extra/export/share/applications --set-key=Exec --set-value=/app/bin/workspacesclient --remove-key=Path --set-icon=com.amazon.Workspaces /app/extra/export/share/applications/com.amazon.Workspaces.desktop"
+          ]
+        },
+        {
+          "type": "extra-data",
+          "filename": "workspacesclient.deb",
+          "only-arches": [ "x86_64" ],
+          "url": "https://d3nt0h4h6pmmc4.cloudfront.net/ubuntu/dists/bionic/main/binary-amd64/workspacesclient_3.0.7.470_amd64.deb",
+          "size": 25105288,
+          "sha256": "046fbeb2977395b9f420b1e2b2f6667d0182d7517848e340395ca943be694fdb",
+          "x-checker-data": {
+              "type": "debian-repo",
+              "package-name": "workspacesclient",
+              "root": "https://d3nt0h4h6pmmc4.cloudfront.net/ubuntu",
+              "dist": "bionic",
+              "component": "main"
+          }
+        },
+        {
+            "type": "file",
+            "path": "com.amazon.Workspaces.appdata.xml"
```

Comment:
New appdata files should use metainfo instead. https://www.freedesktop.org/software/appstream/docs/chap-Quickstart.html

Reply:
Thanks for reviewing. Have pushed a change to move to metainfo.